src/js/router.js

```diff
@@ -5,6 +5,9 @@ import store from "../store";
 
 // Dynamically import the following components so that they are not included in the main build
 // file and are instead asynchrounously loaded when the user goes to the page (to reduce file size of build.js)
+
+const GpaCalculator = () => import("src/components/GpaCalculator/GpaCalculator.vue"); 
```

Comment:
can we give this a [webpack chunk name](https://webpack.js.org/guides/code-splitting/#dynamic-imports), it just makes the build output slightly nicer :)

so something like this:
```suggestion
const GpaCalculator = () => import(/* webpackChunkName: "gpacalculator" */ "src/components/GpaCalculator/GpaCalculator.vue"); 
```